package.json

```diff
@@ -9,6 +9,14 @@
     ".": {
       "types": "./lib/index.d.ts",
       "default": "./lib/index.js"
+    },
+    "./mainApi": {
+      "types": "./lib/api/main/index.d.ts",
+      "default": "./lib/api/main/index.js"
+    },
+    "./basicApi": {
```

Comment:
what do you think about changing ./mainApi to ./main ? (and ./basicApi to ./basic) 🤔